deployments/gpu-operator/values.yaml

```diff
@@ -137,7 +137,11 @@ driver:
     deployDefaultCR: true
     driverType: gpu
     nodeSelector: {}
-  useOpenKernelModules: false
+  kernelModuleType: "auto"
```

Comment:
Let's also update the example ClusterPolicy CR in the OLM bundle: https://github.com/NVIDIA/gpu-operator/blob/55cb4844409c62c84d2f53160702e5ee3b52f3ce/bundle/manifests/gpu-operator-certified.clusterserviceversion.yaml#L44

Reply:
Ah yes, thanks for calling that out